utils/keycodes.js

```diff
@@ -10,3 +20,117 @@ export const DOWN = 40;
 export const DELETE = 46;
 
 export const F10 = 121;
+
+export const ALT = 'alt';
```

Comment:
Every other key exported in this module is its key code, yet for this and other modifiers we export a string? Even though Alt has a code (93)?

Reply:
I think this comes from @tofumatt.

Reply:
I believe it's used to pass to mousetrap and tinymce which wouldn't accept the codes?

Reply:
Oops, I should have at least left a comment. I honestly forget at this point; it could have just been terrible newbie-ness. 😅

Reply:
These are a public API of `@wordpress/keycodes`, which describes exports as:

>Contains keycodes constants for keyboard keys like DOWN, UP, ENTER, etc.

https://www.npmjs.com/package/@wordpress/keycodes

Yet...

```js
import * as keycodes from '@wordpress/keycodes';

keycodes.DOWN // 40 ✅
keycodes.UP // 38 ✅ 
keycodes.ENTER // 13 ✅ 
keycodes.ALT // 'alt' 😱 
```